editor/inspector/editor_inspector.cpp

```diff
@@ -2166,11 +2168,11 @@ void EditorInspectorSection::setup(const String &p_section, const String &p_labe
 
 void EditorInspectorSection::gui_input(const Ref<InputEvent> &p_event) {
 	ERR_FAIL_COND(p_event.is_null());
-	bool has_children_to_show = vbox->get_child_count(false) != 0;
+	bool can_click_unfold = vbox->get_child_count(false) != 0 && !(!checkbox_only && checkable && !checked);
```

Comment:
Uh, double negation. I think the condition can be something like

```suggestion
	bool can_click_unfold = vbox->get_child_count(false) != 0 && (checkbox_only || !checkable || checked);
```

Reply:
In my opinion the double negation is more readable because it is checking against a single case (is how I thought about it), I reorganized it which made the double negation more apparent, I could change it back to:
```suggestion
	bool can_click_unfold = vbox->get_child_count(false) != 0 && !(checkable && !checked && !checkbox_only);
```